app/models/slate_lineup_config.py

```diff
@@ -96,11 +97,13 @@ def get_experiment_from_slate_lineup(slate_lineup_id: str) -> SlateLineupExperim
         return SlateLineupExperimentModel.choose_experiment(slate_lineup_config.experiments)
 
     @staticmethod
-    async def get_slate_configs_from_experiment(experiment: SlateLineupExperimentModel,
-                                                user_id: str = None) -> List[SlateConfigModel]:
+    async def get_slate_configs_from_experiment(
+            slate_lineup_id: str, experiment: SlateLineupExperimentModel,
+            user_id: str = None) -> List[SlateConfigModel]:
```

Comment:
https://docs.python.org/3/library/typing.html#typing.Optional
```suggestion
            user_id: Optiional[str] = None) -> List[SlateConfigModel]:
```